src/index.js

```diff
@@ -31,8 +31,12 @@ export async function run() {
       const tmateReleaseTar = await tc.downloadTool(`https://github.com/tmate-io/tmate/releases/download/${TMATE_LINUX_VERSION}/tmate-${TMATE_LINUX_VERSION}-static-linux-amd64.tar.xz`);
       const tmateDir = path.join(os.tmpdir(), "tmate")
       tmateExecutable = path.join(tmateDir, "tmate")
-      fs.mkdirSync(tmateDir)
+
+      if (fs.existsSync(tmateExecutable))
+        fs.unlinkSync(tmateExecutable)
```

Comment:
Shouldn't this be `rmdirSync()`? Or maybe do both, in case some joker installed a file in that location. Oh, and I would try to catch exceptions to avoid alarming the user (if we both `unlink()` and `rmdir()`, one of them will fail).

Reply:
updated it so only `rmdirSync` gets used. I think a try/catch is not necessary, since it should not happen.